bcipy/helpers/stimuli.py

```diff
@@ -288,13 +293,17 @@ def __call__(self,
         for symbol in symbol_set:
             data_by_targets[symbol] = []
 
-        window_length = 3  # seconds, total length of flickering after prompt for each inquiry
+        buffer = stimulus_duration / 5  # seconds, buffer for each inquiry
```

Comment:
Why `/5`? This will break when changing the duration I think.

Reply:
I added a note right under this line:
`This buffer is used to account for the screen downtime between each stimulus. There seems to be a duty cycle of 80% for the stimuli, so we add a buffer of 20% of the stimulus length.` 
I'm not sure why this "duty cycle" occurs, but it's something that we noticed in the Triggers file – the total time between symbol flashes were `0.30 sec` even though the flash time is set as `time_flash = 0.25 sec` in the parameters file of each experiment. So yes, if the duration changes this buffer length will change too (in that case I hope my 80/20 calculation is correct!). 

Fyi, I use this parameter just to calculate the inquiry length:
```python
window_length = (stimulus_duration + buffer) * num_stimuli_per_inquiry
``` 
Instead, I can maybe just calculate the longest inquiry length with a formula similar to the one in InquiryReshaper: 
```python
        def get_inquiry_len(inq_trigs):
            return inq_trigs[-1] - inq_trigs[0]

        longest_inquiry = max(grouper(triggers, trials_per_inquiry, fillvalue='x'), key=lambda xy: get_inquiry_len(xy))
```